src/components/ChecWysiwyg.vue

```diff
@@ -0,0 +1,571 @@
+<template>
+  <div class="wysiwyg" :class="classNames">
+    <EditorContent
+      :editor="editor"
+      v-bind="sharedInputProps"
+      v-on="nonInputListeners"
+      @input="handleInput"
+    />
+    <label
+      v-if="label"
+      class="wysiwyg__label"
+      :data-content="label"
+      :for="id"
+    >
+      {{ label }}
+      <span
+        v-if="required"
+        class="wysiwyg__required-text"
+      >
+        {{ $t('general.required') }}
+      </span>
+    </label>
+    <EditorMenuBar v-slot="{ commands, isActive }" :editor="editor">
+      <div class="wysiwyg__menubar">
+        <button
+          class="wysiwyg__menubar--button"
+          :class="{ 'wysiwyg__menubar--button--is-active': isActive.bold() }"
+          @click="commands.bold"
+        >
+          <ChecIcon icon="bold" size="base" />
+        </button>
+        <button
+          class="wysiwyg__menubar--button"
+          :class="{ 'wysiwyg__menubar--button--is-active': isActive.italic() }"
+          @click="commands.italic"
+        >
+          <ChecIcon icon="italic" size="base" />
+        </button>
+        <button
+          class="wysiwyg__menubar--button"
+          :class="{ 'wysiwyg__menubar--button--is-active': isActive.heading({ level: 1 }) }"
+          @click="commands.heading({ level: 1 })"
+        >
+          H1
```

Comment:
Any reason these aren't icons?

Reply:
They weren't icons in the design so I kept it that way.